### PR TITLE
41424 Storybook Introduction Page Links

### PR DESCRIPTION
**Pull request by pdavies88**
## Chromatic

https://41424-sb-intro-page--60f9b557105290003b387cd5.chromatic.com

## Description
Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/41425

## Screenshots
Screen Shot 2022-06-13 at 11 26 01 AM

## Acceptance criteria
- [X] Sections and links are created for the pages listed above on the [Introduction page](https://design.va.gov/storybook/?path=/docs/about-introduction--page)

## Definition of done
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/storybook/stories/introduction.stories.mdx [hunk 1]
```diff
@@ -22,3 +22,62 @@ Enjoy!
...
+How you implement and install the design system on your project.
+
+<strong>
+  <a
+    href="https://design.va.gov/about/developers#using-the-design-system"
+    target="_blank"
+  >
+    Link Here
+  </a>
+</strong>
```
Comment (k80bowman) [inline packages/storybook/stories/introduction.stories.mdx:37]:
```suggestion
<strong>
  <a
    href="https://design.va.gov/about/developers#using-the-design-system"
    target="_blank"
  >
    Implement and install the design system on your project.
  </a>
</strong>
```

Not sure about what this looks like with the `<strong>` tag, but check it out and see what you think. I'm not keen on the "link here" text, I'd rather have the links in context. I also prefer declarative statements here, so please update the following two to match this one.

File: packages/storybook/stories/introduction.stories.mdx [hunk 2]
```diff
@@ -22,3 +22,62 @@ Enjoy!
...
+
+<strong>
+  <a href="https://design.va.gov/about/feedback" target="_blank">
+    Link Here
+  </a>
+</strong>
+
+# Maturity Scale
+
+The maturity scale indicates the level of maturity of each component and pattern in the system. It helps designers and developers know if they can use the component and what level of caution they should have when doing so.
```
Comment (k80bowman) [inline packages/storybook/stories/introduction.stories.mdx:77]:
```suggestion
The maturity scale indicates the level of maturity of each component and pattern in the system. It helps designers and developers know if they should use the component and what level of caution they should have when doing so.
```

**Review by k80bowman — CHANGES_REQUESTED** [review]
Please see comments above.

**Comment by k80bowman** [comment]
Thank you for making the updates, Preston. Just one more question, why do the new links look different than the "how to use Storybook" link? If there is a way to make these new ones look like that one (with the darker blue and the underline) that would be great.

image

**Comment by k80bowman** [comment]
@andresriveratoro @pdavies88 It looks like that issue was taken care of in #437  (as noted in the ticket). Preston, if this fix did not fix the links in this PR, it looks like some additional work may be necessary.

**Comment by pdavies88** [comment]
@k80bowman I added some CSS to `.sbdocs-a` which gets generated when creating storybook pages with .mdx so now all links look the same. Screenshot here:
Screen Shot 2022-06-13 at 12 28 07 PM

**Pull request merged**